modules/ROOT/pages/sharing-custom-configuration-fragments.adoc

```diff
@@ -0,0 +1,83 @@
+= Sharing Custom Configuration Fragments
```

Comment:
We need to redo this page unfortunately. We need to create a new doc page with the content of this KB https://help.mulesoft.com/s/article/How-to-add-a-call-to-an-external-flow-in-Mule-4.